botocore/data/autoscaling/2011-01-01/waiters-2.json

```diff
@@ -0,0 +1,31 @@
+{
+  "version": 2,
+  "waiters": {
+    "LifecycleStateInService": {
+      "operation": "DescribeAutoScalingInstances",
+      "maxAttempts": 40,
+      "delay": 15,
```

Comment:
This seemed to be the default in EC2, can change, I guess Autoscaling rate limits might be lower?